signup.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="utf-8">
+    <title>회원가입</title>
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css">
+    <link rel="stylesheet" href="styleLogin.css">
+  </head>
+  <body>
+    <div class="content">
+      <div class="head-img">
+        <a class="login-link" href="/">
+          <img class="pandaMarketImg" src="img/logo.png">
+        </a>
+      </div>
+      <div>
+        <p>이메일</p> 
+        <input type="email" name="userEmail" id="userName">
```

Comment:
❗️ 수정요청
해당 태그는 input을 설명하는 역할이므로 label 태그를 사용하시는 것이 좋습니다~
또한 해당 label 태그가 어떤 요소를 설명하는 지 알 수 있도록 연결해주시는 것이 좋습니다.

```suggestion
        <label for="userName">이메일</label> // label 태그에 for 속성에 연결할 input 태그의 id를 넣어주세요~ 
        <input type="email" name="userEmail" id="userName">
```